tests/integration/test_github.py

```diff
@@ -403,6 +403,13 @@ def test_project_column(self):
 
         assert isinstance(r, github3.projects.ProjectColumn)
 
+    def test_public_gists(self):
+        """Test the ability to iterate over the public gists."""
+        cassette_name = self.cassette_name('public_gists')
+        with self.recorder.use_cassette(cassette_name):
+            for r in self.gh.public_gists(since='2018-07-13', number=25):
+                assert isinstance(r, github3.gists.ShortGist)
```

Comment:
I wonder if it makes sense to assert that all of these have a `created_at` after `2018-07-13`.

Reply:
I think it's a good idea, although I believe (if I read the [documentation](https://developer.github.com/v3/gists/#list-all-public-gists) correctly) that it's `updated_at`.